visidata/column.py

```diff
@@ -202,16 +209,19 @@ def fmtstr(self, v):
 
     @property
     def formatter(self):
-        return self._formatter.__name__[7:]  # remove leading 'format_'
+        return self._formatter  # remove leading 'format_'
 
     @formatter.setter
     def formatter(self, v: str):
-        self._formatMaker = getattr(col, 'format_'+(v or 'generic'))
-        self._formatter = col._formatMaker(self._format)
+        self._formatMaker = getattr(self, 'format_'+(v or 'generic'))
+        self._formatter = self._formatMaker(self._formatdict)
 
     def format_generic(self, fmtstr):
         return self.formatValue
 
+    def format(self, *args, **kwargs):
+        return self.formatter(*args, **kwargs)
```

Comment:
`cursorCol.format` not `cursorCol.formatter` was what was called everywhere. I think we want `cursorCol` to be returning `formatter`. Otherwise, we always use `formatValue` which uses the type's formatting, and `formatter` is never called.

Kept the default to `formatValue`.